blockchain/projects_test.go

```diff
@@ -620,16 +504,16 @@ func Test_DeployContract(t *testing.T) {
 				}
 			}`
 
-		accounts, err := projects.CreateInitialAccounts(proj.ID)
+		_, err := projects.CreateInitialAccounts(proj.ID)
 		assert.NoError(t, err)
 
-		accA, err := projects.DeployContract(proj.ID, accounts[0].Address, scriptA)
+		deploy1, err := projects.DeployContract(proj.ID, model.NewAddressFromString("0x05"), scriptA)
 		require.NoError(t, err)
-		assert.Equal(t, accA.DeployedCode, scriptA)
+		assert.Equal(t, deploy1.Title, "HelloWorldA")
 
-		accB, err := projects.DeployContract(proj.ID, accounts[1].Address, scriptB)
+		deploy2, err := projects.DeployContract(proj.ID, model.NewAddressFromString("0x06"), scriptB)
```

Comment:
Why is the account address no longer referenced but hardcoded by value? If this can be avoided it's better since the tests are more robust if changing values.

Reply:
I'll change this back. It happened previously because I initially removed the accounts and thus didn't have an account to reference.

Reply:
Done